browsers/README.md

```diff
@@ -28,13 +28,14 @@
 [cypress/browsers:node12.14.1-chrome85-ff81](./node12.14.1-chrome85-ff81) | `cypress/base:12.14.1` | `85.0.4183.121` | `81.0`
 [cypress/browsers:node14.10.1-edge88](./node14.10.1-edge88) | `cypress/base:14.10.1` | 🚫 | 🚫 | `88.0.673.0 dev`
 [cypress/browsers:node14.15.0-chrome86-ff82](./node14.15.0-chrome86-ff82) | `cypress/base:14.15.0` | `86.0.4240.193` | `82.0.3`
-[cypress/browsers:node14.15.0-chrome96-ff94](./node14.15.0-chrome96-ff94) | `cypress/base:14.15.0` | `96.0.4664.45` | `94.0.2`
```

Comment:
Could you add this line back? It's an issue with the generator script and we're currently working on it. Thanks!

Reply:
Done @admah